cmd/version/version.go

```diff
@@ -78,8 +82,48 @@ func NewCmdVersion(rootCmd *cobra.Command) *cobra.Command {
 			}
 
 			util.TablePrinter(util.VersionColumnDefinition, rows, printers.PrintOptions{Wide: true})
+			checkForLatestVersion(rootCmd.Version)
 		},
 	}
 	cmd.SetUsageTemplate(versionCmdHelp)
 	return cmd
 }
+
+func checkForLatestVersion(currVersion string) {
+	// getting the latest version of openebsctl from sigs.k8s.io/krew-index
+	resp, err := http.Get("https://raw.githubusercontent.com/kubernetes-sigs/krew-index/master/plugins/openebs.yaml")
+	util.CheckErr(err, util.Fatal)
+	defer resp.Body.Close()
+
+	body, err := ioutil.ReadAll(resp.Body)
+	util.CheckErr(err, util.Fatal)
```

Comment:
return err if err is not nil, though this will likely not be the case unless `resp.Code != 200` i.e. the response to this request is `HTTP 200 OK`